jvb/src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -1044,6 +1044,8 @@ public void relayExpired(Relay relay)
     {
         String id = relay.getId();
         relaysById.remove(id);
+
+        endpointsById.forEach((i, senderEndpoint) -> senderEndpoint.removeReceiver(id));
```

Comment:
With a full-mesh we should only ever be doing `Endpoint.addReceiver` for local endpoins. Should we enforce it and change this to iterate over localEndpoints?